packages/fuel-gauge/src/e2e-script.test.ts

```diff
@@ -1,29 +1,79 @@
 /* eslint-disable no-console */
-import { Provider, TransactionType, WalletUnlocked } from 'fuels';
+import {
+  DEVNET_NETWORK_URL,
+  TESTNET_NETWORK_URL,
+  Provider,
+  TransactionType,
+  WalletUnlocked,
+  CHAIN_IDS,
+  rawAssets,
+  assets,
+} from 'fuels';
 
 import { getScript } from './utils';
 
+enum Networks {
+  DEVNET = 'devnet',
+  TESTNET = 'testnet',
+}
+
+type ConfiguredNetwork = {
+  networkUrl: string;
+  privateKey?: string;
+  faucetUrl: string;
+  txIds?: {
+    [TransactionType.Mint]: string;
+    [TransactionType.Upgrade]: string;
+    [TransactionType.Upload]: string;
+  };
+};
+
+const configuredNetworks = {
+  [Networks.DEVNET]: {
+    networkUrl: DEVNET_NETWORK_URL,
+    privateKey: process.env.DEVNET_WALLET_PVT_KEY,
+    faucetUrl: `https://faucet-devnet.fuel.network/`,
+    txIds: {
+      [TransactionType.Upgrade]:
+        '0xe2c03044fe708e9b112027881baf9f892e6b64a630a629998922c1cab918c094',
+      [TransactionType.Upload]:
+        '0x94bc2a189b8211796c8fe5b9c6b67624fe97d2007e104bf1b30739944f43bd73',
+    },
+  } as ConfiguredNetwork,
+  [Networks.TESTNET]: {
+    networkUrl: TESTNET_NETWORK_URL,
+    privateKey: process.env.TESTNET_WALLET_PVT_KEY,
+    faucetUrl: `https://faucet-testnet.fuel.network/`,
+    txIds: {
+      [TransactionType.Upgrade]:
+        '0xd64e3f7589bc1c6dcf1e419f4a3a8fc21d3694abf98f151000f34682d1cacdce',
+      [TransactionType.Upload]:
+        '0x996eec87a702ac978663fe67dbde7ab94d31f32b1860fbfc527d4b5447b3446c',
+    },
+  } as ConfiguredNetwork,
+};
+
+const selectedNetworks: Networks[] = [Networks.DEVNET];
```

Comment:
According to what we discussed, the PR must pass against both networks before we merge it. Correct?

```suggestion
const selectedNetworks: Networks[] = [Networks.DEVNET, Networks.TESTNET];
```